aten/src/ATen/native/quantized/cpu/qnnpack/include/conv_utils.h

```diff
@@ -29,43 +29,46 @@ static inline size_t compute_output_dimension(
 }  // namespace
 
 struct conv_param_t {
-  const std::array<uint32_t, 2> kernel_dims; // kernel width, height
-  const std::array<uint32_t, 2> stride_dims; // subsampling width, height
-  const std::array<uint32_t, 2> dilation; // dilation width, height
-  const std::array<uint32_t, 4> padding; // input padding top, left, bottom, right
-  const std::array<uint32_t, 2> adjustment_dims; // output adjustment
+  std::array<uint32_t, 2> kernel_dims; // kernel width, height
+  std::array<uint32_t, 2> stride_dims; // subsampling width, height
+  std::array<uint32_t, 2> dilation; // dilation width, height
+  std::array<uint32_t, 4> padding; // input padding top, left, bottom, right
+  std::array<uint32_t, 2> adjustment_dims; // output adjustment
 
-  const uint32_t groups;
-  const size_t input_channels;
-  const size_t output_channels;
-  const uint8_t kernel_zero_point;
-  const float kernel_scale;
-  const uint8_t output_min;
-  const uint8_t output_max;
-  const bool transpose;
+  uint32_t groups;
+  size_t input_channels;
+  size_t output_channels;
+  uint8_t* kernel_zero_points{nullptr};
+  float* requantization_scales{nullptr};
+  uint8_t output_min;
+  uint8_t output_max;
+  bool transpose;
+  bool is_initialized;
```

Comment:
Why remove the const qualifiers here?

Reply:
So with const qualifiers I could not construct conv_param_t like `conv_p = conv_param_t(...` since it generates rvalues ref and default move assign cannot be used on consts.